graylog2-web-interface/src/components/event-definitions/event-definition-form/NotificationSettingsForm.jsx

```diff
@@ -70,6 +81,15 @@ class NotificationSettingsForm extends React.Component {
               Events with keys have different grace periods for each key.
             </HelpBlock>
           </FormGroup>
+          <Input type="number"
+                 id="backlogsize"
+                 name="backlog_size"
+                 label="Message Backlog"
+                 required
+                 onChange={this.handleBacklogSizeChange}
+                 help="The maximum number of backlog messages to be included in notifications. Use 0 to disable."
```

Comment:
I think we should disable the backlog with a checkbox, not using 0. We could use something like this, if we want to put everything in a single input: https://5c507d49471426000887a6a7--react-bootstrap.netlify.com/components/forms/#forms-input-addons